packages/apps/proof-of-us/src/components/Status/ErrorStatus.tsx

```diff
@@ -1,14 +1,16 @@
 import { MonoClose } from '@kadena/react-icons';
 import { Stack } from '@kadena/react-ui';
+import Link from 'next/link';
 import type { FC, PropsWithChildren } from 'react';
 import { Button } from '../Button/Button';
 import { MessageBlock } from '../MessageBlock/MessageBlock';
 import { Heading } from '../Typography/Heading';
 
 interface IProps extends PropsWithChildren {
-  handleClose?: () => void;
+  handleClose?: string;
```

Comment:
if we make it a url, maybe better for readability to rename.
`handleClose` kind of implies a function.
maybe `closeUrl` or something